packages/server/src/alerts/alerts.service.ts

```diff
@@ -19,18 +26,60 @@ export class AlertsService {
           const payload = alert.payload as RephraseQuestionPayload;
           const question = await QuestionModel.findOne(payload.questionId);
 
-          const queue = await QueueModel.findOne(payload.queueId);
-          if (question.closedAt || !(await queue.checkIsOpen())) {
-            console.log(`Rephrase Question alert with id ${alert.id} expired`);
+          const queue = await QueueModel.findOne(payload.queueId, {
```

Comment:
minor: Why do we need to load the stafflist here? Is it to check if the queue is open?

Reply:
cc: @IrisLiu-00  
iirc, i think it had smth to do with fetching correctly. related to this pr:https://github.com/sandboxnu/office-hours/pull/834